### PR TITLE
:seedling: Build virtualbmc image using python image

### PR DESCRIPTION
**Pull request by elfosardo**
Switch the virtualbmc image to python base image based on bookworm to reduce its size from 400MB to 290MB

### CODE REVIEW

File: resources/vbmc/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,13 @@
-ARG BASE_IMAGE=quay.io/centos/centos:stream9
+FROM docker.io/library/python:3.9.18-slim-bookworm
```
Comment (tuminoid) [inline resources/vbmc/Dockerfile:1]: Can we keep the BASE_IMAGE arg as it used to be, but defaulting to this python image?

File: resources/vbmc/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-  curl https://trunk.rdoproject.org/centos9-master/puppet-passed-ci/delorean.repo -o /etc/yum.repos.d/rdo.repo && \
-  curl https://trunk.rdoproject.org/centos9-master/delorean-deps.repo -o /etc/yum.repos.d/rdo-deps.repo && \
-  dnf install -y python3-virtualbmc && \
-  dnf clean all && \
-  rm -rf /var/cache/{yum,dnf}/*
+RUN apt-get update && \
+    apt-get install -y libvirt-dev ssh gcc && \
+    apt-get clean && \
+    pip3 install --no-cache-dir \
+        virtualbmc==${VIRTUALBMC_VERSION} && \
```
Comment (tuminoid) [inline resources/vbmc/Dockerfile:10]:
```suggestion
        virtualbmc=="${VIRTUALBMC_VERSION}" && \
```

File: resources/vbmc/Dockerfile (hunk 1 above)
Comment (elfosardo) [reply resources/vbmc/Dockerfile:1]: yes, I think I copy-pasted witout realizing I was removing it!

**Review by tuminoid — COMMENTED** [review]
/lgtm

Good improvement!

**Comment by elfosardo** [comment]
thanks @tuminoid 
had to update the path of vbmcd

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**